assets/permissions.py

```diff
@@ -76,64 +162,38 @@ def _validate_asset_user_institution(user, department):
         """Validates that the user is member of the department that the asset belongs to
         (asset_department)."""
 
-        if user is None or user.is_anonymous or department is None:
-            return False
-
         lookup_response = get_person_for_user(user)
+        if lookup_response is None:
```

Comment:
This check is unnecessary since ``get_person_for_user`` raises if the user has no lookup response.

Reply:
Not a blocker, though, just something which explains the codecov complaints.